cocoapods.rb

```diff
@@ -14,8 +16,11 @@ def use_unimodules!(custom_options = {})
   unimodules = {}
   unimodules_duplicates = []
 
+  project_directory = find_project_directory()
```

Comment:
Why don't we use `Pod::Config.instance.project_root` (vel `.installation_root`)?